docs/zdgbook/ObjectPublishing.rst

```diff
@@ -392,15 +392,21 @@ about to be traversed.
 Traversal and Acquisition
 -------------------------
 
-Acquisition affects traversal in several ways. See Chapter 5,
-"Acquisition" for more information on acquisition. The most obvious
-way in which acquisition affects traversal is in locating the next
-object in a path. As we discussed earlier, the next object during
-traversal is often found using 'getattr'. Since acquisition affects
-'getattr', it will affect traversal. The upshot is that when you are
-traversing objects that support implicit acquisition, you can use
-traversal to walk over acquired objects. Consider the object
-hierarchy rooted in 'fruit'::
+.. note::
+
+  Simply put, acquistion means, that a Zope object can aquire any
```

Comment:
Typo "acquistion", and the second comma needs to go

Reply:
Thanks a lot! I pushed the changes + one more typo fix and finally installed a spell checker for my editor.